docs/docs/10.8-pure-render-mixin.md

```diff
@@ -6,6 +6,8 @@ prev: update.html
 next: perf.html
 ---
 
+> Note: React 15.3 added the [React.PureComponent](https://github.com/facebook/react/releases/tag/v15.3.0) base class which is a more convenient way to implement the behavior below that does not rely upon mixins.
+
```

Comment:
I think we need to be explicit about how `React.PureComponent` is for ES6 classes specifically. I like how @jimfb [phrased it](https://github.com/facebook/react/pull/7465#discussion_r74331613). I know we say "base class" here, so it implies it's for ES6 classes, but maybe we can just make that clearer?